### ISSUE TITLE
Create collaboration group(s) in Portal to connect with orgs/agencies

### ISSUE THREAD

**Issue by gregbunce** [agrc/planning-queue#284]
### Benefit

This will allow UGRC to better collaborate with the data stewards as well as our data contributors - allowing for more efficient, timely data updates. 

There are still some unknowns as to how this will work so we need to build a test DB and begin testing with a willing party (WFRC) to work out the details and kinks.

### Acceptance Criteria

- [x] invite appropriate collaborators via invitations
- [x] test collaboration between UGRC Portal and WFRC AGOL
- [x] Continue developing governance

### Notes

Let's test collaboration with WFRC on roads and trails and pathways

#### Greg

- none: we're testing

#### Steve

- tests can turn into production, so let's be careful here.
- the governance hasn't been solidified yet

### Issue Reference

refs #283

**Comment by MichaelAGRC** [comment]
**Sprint 4 Notes**

Have meeting scheduled with Esri (Will and Dave) on Tuesday, February 20th at 11am.  Greg has been invited as well to cover some collaboration questions in tandem with the branch versioning assistance.

**Comment by gregbunce** [comment]
I met with WFRC today and did some preliminary testing.  We made partial progress by creating a collaboration workspace and group.  We're scheduled to meet again to see if we can share test data.

**Comment by gregbunce** [comment]
progress was made on this quarter. i have been working with wfrc on distributed collaboration. we made enough progress that we were able to share content and edit content, however the content that wfrc edits is not syncing back to our Portal. 
I met with Will from ESRI a few times to see if we could work it out and we determined that it might be a bug. on march 18 we determined that it's best to put in a ticket with esri and have them take a look. michael will be leading that up and Will be watching it and elevating it to the appropriate levels at esri. 
as far as this feature: we successfully created collaborations groups and connected our orgs (wfrc and ugrc).
